Update howto-authentication-passwordless-security-key-windows.md (#1090)
MicrosoftDocs/entra-docs: docs/identity/authentication/howto-authentication-passwordless-security-key-windows.md

### Manage security key biometric, PIN, or reset security key

* Windows 10 version 1903 or higher

>    * Users can open **Windows Settings** on their device > **Accounts** > **Sign-in options** > **Security Key** then click the **Manage** button.

Reviewer: ```suggestion
   * On your device, go to **Windows Settings** > **Accounts** > **Sign-in options** > **Security Key**, and then select the **Manage** button.
```
